### PR TITLE
Inform router when devices and associated resources are updated

### PR DESCRIPTION
**Pull request by vicmgs**

### CODE REVIEW

File: lib/console/multi_buys/multi_buys.ex [hunk 1]
```diff
@@ -25,4 +27,29 @@ defmodule Console.MultiBuys do
     |> MultiBuy.changeset(attrs)
     |> Repo.update()
   end
+
+  def get_all_multi_buy_associated_device_ids(id) do
+    device_ids =
+      get_associated_multi_buy_devices(id)
```
Comment (ambuznego) [inline lib/console/multi_buys/multi_buys.ex:33]: can we just do a select of ids so we don't have to map over the results?

Comment (vicmgs) [reply lib/console/multi_buys/multi_buys.ex:33]: good idea!

Comment (ambuznego) [reply lib/console/multi_buys/multi_buys.ex:33]: though if it gives you `[%{ id: 'someid' }, %{ id: 'someotherid' }]`, you may have to iterate anyway?

Comment (ambuznego) [reply lib/console/multi_buys/multi_buys.ex:33]: maybe  you could use `uniq_by` in that case?

File: lib/console/multi_buys/multi_buys.ex [hunk 2]
```diff
@@ -25,4 +27,29 @@ defmodule Console.MultiBuys do
...
+
+    label_device_ids =
+      get_associated_multi_buy_labels(id)
```
Comment (ambuznego) [inline lib/console/multi_buys/multi_buys.ex:36]: probs same with this, using select

**Review by ambuznego — APPROVED** [review]
lgtm

**Pull request merged**